base/interactiveutil.jl

```diff
@@ -51,10 +51,7 @@ function edit(path::AbstractString, line::Integer=0)
         cmd = line != 0 ? `$command $path:$line` : `$command $path`
     elseif startswith(name, "notepad++")
         cmd = line != 0 ? `$command $path -n$line` : `$command $path`
-    elseif is_windows() && (name == "start" || name == "open")
-        cmd = `cmd /c start /b $path`
-        line_unsupported = true
```

Comment:
I don't see why this is redundant? shouldn't you still get the line number warning?